common-features/data-binding/descriptors.md

```diff
@@ -0,0 +1,317 @@
+---
+title: Descriptors - FilterDescriptor, SearchFilter, SortDescriptor, GroupDescriptor
+page_title: Descriptors - FilterDescriptor, SearchFilter, SortDescriptor, GroupDescriptor
+description: Discover the FilterDescriptor, SortDescriptor, SearchFilter and GroupDescriptor properties and how to access their values.
+slug: common-features-descriptors
+tags: telerik,blazor,filterdescriptor, sortdescriptor, groupdescriptor, searchfilter
+published: True
+position: 10
+---
+
+
+## Components with Descriptors
+
+This article explains how to retrieve the applied filtering, searching, sorting, and grouping criteria in Blazor components. The article applies to components that support these features. The components that offer one or all of the functionalities are:
+* The [Filter]({%slug filter-overview%})
+* The [Gantt]({%slug gantt-overview%})
+* The [TreeList]({%slug treelist-overview%})
+* All components that [expose the `OnRead` event]({%slug common-features-data-binding-onread%}#components-with-onread-event), excluding the [ListView]({%slug listview-overview%}), because the ListView doesn't support built-in filtering, searching, sorting, and grouping.
```

Comment:
Sort by importance.

```suggestion

* [Components that [expose an `OnRead` event]({%slug common-features-data-binding-onread%}#components-with-onread-event), excluding the [ListView]({%slug listview-overview%}), because the ListView doesn't support built-in filtering, searching, sorting, and grouping.
* [Filter]({%slug filter-overview%})
* [Gantt]({%slug gantt-overview%})
* [TreeList]({%slug treelist-overview%})
```